feedback-module/src/components/Module.js

```diff
@@ -119,12 +120,40 @@ function Module({ pageTitle, endpoint, dir }) {
     return checkedFields.some((field) => field.checked);
   };
 
+  //Check if valid email address
+  const invalidEmail = (email, required) => {
+    const re = /^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/;
+    let valid = re.test(email);
+    if (!required && email === "") {
+      valid = true;
+    }
+    return !valid;
```

Comment:
hmmm not a change in functionality but could we shorten it to
```
if (!required && email === "") return false;
return !re.test(email);
```

Reply:
done